src/assets/data/edition/series/1/section/2/op27/textcritics.json

```diff
@@ -54,7 +54,7 @@
                     "svgGroupId": "g7880",
                     "measure": "Bl. 1<sup>r</sup>",
                     "system": "1b",
-                    "position": "G<sub>es</sub>: 10.–11. Reihenton",
```

Comment:
`10.-11. Reihenton` was correct, wasn't it?

Reply:
ofc, thanks!